projects/netgrif-components/src/lib/side-menu/content-components/user-impersonate/user-impersonate.component.html

```diff
@@ -1,6 +1,6 @@
 <mat-toolbar color="primary">{{'side-menu.impersonation.title' | translate}}</mat-toolbar>
```

Comment:
Could you please add the following style to the menu?

-webkit-user-select: none; 
  -ms-user-select: none; 
  user-select: none;

Reply:
done